source/public/index.html

```diff
@@ -45,28 +53,40 @@ <h1>Welcome back to your developer journal!</h1>
         </section>
 
         <section id="projects">
-            <h2>Ongoing Projects</h2>
+            <h2>Projects</h2>
+                 <button class="newProjectButton" title="new project">
+                    + New Project
+                </button>
             <ul id="project-list">
-                <li class="project-item">
+                <!--li--><ul class="project-item">
                     <div class="project-details">
                         <label class="project-name">Project Name</label>
+                         <div class="project-btns">
+                        <button class="edit-project">
+                            <i class="fa-solid fa-pencil"></i>
+                        </button>
+                        <button class="project-item-delete" title="Delete project">
+                            <i class="fa-solid fa-trash"></i>
+                        </button> 
+                    </div> 
                         <p class="project-date">Deadline: May 30, 2024</p>
-                        <label class="project-priority">High</label>
+                        <label class="project-priority">Priority: <span class="prioritytype">High</span></label>
```

Comment:
Could you also change the class name here to `priority-type` for case consistency?

Reply:
Actually, could you change it to `high` (eventually with JS integration there should be high, medium, and low tags - could you also add the styling for those classes with the different colors?